cmd/integrations/export.go

```diff
@@ -38,13 +38,17 @@ var ExportCmd = &cobra.Command{
 			return err
 		}
 
-		return integrations.Export(folder)
+		// check if threads argument was passed, use default value if not
+		threads, _ := cmd.Flags().GetInt("threads")
```

Comment:
You will always get `3` here, since that is the default. Maybe you should set default to -1 and -1 means we don't use goroutines or you need another boolean flag.

Reply:
Yes, by default I did set 3, but I like the idea to keep it as -1.. and making it non-goroutine based download